### PR TITLE
Add Quote component

### PR DESCRIPTION
**Pull request by jasperswart**
desktop

For mobile:
mobile

Before opening a pull request, please ensure:

- [ ] You've added or updated the README.mdx of the story of the component
- [x] You have been following the guidelines, written in the [README](../blob/master/docs/CONTRIBUTING.md#user-content-conventions-and-rules) file
- [x] Your code has the necessary tests written
- [x] You've exported the component in [index.ts](./packages/asc-ui/src/index.ts)
- [x] You have updated the [CHANGELOG.md unreleased sections](../blob/master/CHANGELOG.md#muser-content-unreleased)

### CODE REVIEW

File: packages/asc-ui/src/components/Quote/QuoteStyle.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  @media screen and ${breakpoint('max-width', 'tabletS')} {
+    padding-right: 15px;
+
+    & svg {
+      width: 20px;
+      height: 20px;
+      flex-basis: 35px;
+    }
+  }
+`
```
Comment (jonkoops) [inline packages/asc-ui/src/components/Quote/QuoteStyle.ts:36]: Perhaps we can replace some of these units with their `themeSpacing()` equivalent?

File: packages/asc-ui/src/index.ts [hunk 2]
```diff
@@ -75,6 +75,7 @@ import {
 import Modal from './components/Modal'
 import Paragraph, { ParagraphStyle } from './components/Paragraph'
 import Portal from './components/Portal'
+import Quote, { QuoteStyle } from './components/Quote'
```
Comment (jonkoops) [inline packages/asc-ui/src/index.ts:78]: Perhaps we can use a `export { default as Quote, QuoteStyle }` statement here?

**Review by jonkoops — APPROVED** [review]
Very nice, thanks for building this @jasperswart!

File: packages/asc-ui/src/components/Quote/QuoteStyle.ts (hunk 1 above)
Comment (jasperswart) [reply packages/asc-ui/src/components/Quote/QuoteStyle.ts]: gedaan

File: packages/asc-ui/src/index.ts (hunk 2 above)
Comment (jasperswart) [reply packages/asc-ui/src/index.ts:78]: gedaan

**Pull request merged**